spec/reek/smells/control_parameter_spec.rb

```diff
@@ -270,7 +270,7 @@ def things(arg)
         end
       EOS
       ctx = Reek::Context::MethodContext.new(nil, Reek::Source::SourceCode.from(src).syntax_tree)
-      smells = detector.examine(ctx)
+      smells = detector.run_for(ctx)
```

Comment:
>> Perhaps removing that indirection will be something to do in a next refactoring.

Absolutely. I have this on my todo list already, but I wanted to keep this PR manageable :)